config/routes.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.routes.draw do
+  resources :users, only: [:new, :create, :edit]
```

Comment:
## Add update to the routes 
* [ ] `resources :users, only: [:new, :create, :edit, :update]` in config/routes.rb ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_route1.png) [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_route2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))
```suggestion
  resources :users, only: [:new, :create, :edit, :update]
```

Reply:
:update route has been added.